proxy-robots-module/src/main/java/orwell/proxy/config/ConfigFactory.java

```diff
@@ -17,7 +17,7 @@ public ConfigFactory(final ConfigFactoryParameters configFactoryParameters) {
         logback.debug("IN");
         final Configuration configuration = new Configuration(configFactoryParameters);
 
-        if (false == configuration.isPopulated) {
+        if (!configuration.isPopulated) {
```

Comment:
It might be easier to read without the negation.
Does it make sense to have all those objects set to null ?